vital/config/config_block_io.h

```diff
@@ -236,16 +238,60 @@ void VITAL_CONFIG_EXPORT write_config( config_block_sptr const& config,
  *   The application version number, used to build the list of standard
  *   locations to be searched.
  * \param install_prefix
+ *   The prefix to which the application and KWIVER are installed (should be
+ *   one directory higher than the location of the executing binary).
+ *
+ * \return
+ *   List of additional application configuration search paths.
+ */
+config_path_list_t VITAL_CONFIG_EXPORT
+application_config_file_paths(std::string const& application_name,
+                              std::string const& application_version,
+                              config_path_t const& install_prefix);
+
+// ------------------------------------------------------------------
+/// Get additional application configuration file paths
+/**
+ * This includes the KWIVER_CONFIG_PATH contents at the head of the returned
+ * list.  The paths returned include application config paths followed by
+ * KWIVER config paths for the current version of KWIVER.
+ *
+ * \param application_name
+ *   The application name, used to build the list of standard locations to be
+ *   searched.
+ * \param application_version
+ *   The application version number, used to build the list of standard
+ *   locations to be searched.
+ * \param app_install_prefix
  *   The prefix to which the application is installed (should be one directory
  *   higher than the location of the executing binary).
+ * \param kwiver_install_prefix
+ *   The prefix to which the KWIVER is installed.
  *
  * \return
  *   List of additional application configuration search paths.
  */
 config_path_list_t VITAL_CONFIG_EXPORT
 application_config_file_paths(std::string const& application_name,
                               std::string const& application_version,
-                              config_path_t const& install_prefix);
+                              config_path_t const& app_install_prefix,
+                              config_path_t const& kwiver_install_prefix);
+
+// ------------------------------------------------------------------
+/// Get KWIVER configuration file paths
+/**
+ * This includes the KWIVER_CONFIG_PATH contents at the head of the returned
```

Comment:
Either your separator line is too short, or this is too long. (BTW, I have not rigorously checked line lengths throughout...)

Reply:
The separators in this files are all length 70.  What did we decide on? 80?

Reply:
79 (if you don't want to count, copy any of the ones from vital/ranges; those are "correct" :slightly_smiling_face:). It's actually not a big deal if you'd rather ignore it, just that it "threw" me a bit in review because the separator lines (when they are the "right" length) make a convenient gauge for line length, and it's a little odd seeing text that is longer.